aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -214,40 +214,45 @@ void TensorIterator::allocate_outputs() {
 }
 
 #ifdef BUILD_NAMEDTENSOR
-void TensorIterator::propagate_names_to_outputs() {
-  bool should_perform_name_inference = std::any_of(
+void TensorIterator::compute_names() {
+  bool should_infer_names = std::any_of(
```

Comment:
I see you renamed this variable, and then removed the usage of the prior name, ... is this used somewhere?

Reply:
Oh, that was an accident. It should be used right after. Thanks for catching